src/inference/tests/functional/ov_extension_test.cpp

```diff
@@ -353,4 +358,14 @@ TEST_F(OVExtensionTests, load_old_extension) {
 TEST_F(OVExtensionTests, load_incorrect_extension) {
     EXPECT_THROW(core.add_extension(getIncorrectExtensionPath()), ov::Exception);
 }
+
+TEST_F(OVExtensionTests, load_relative_path) {
+    try {
+        core.add_extension(getOVExtensionRelativePath());
+        SUCCEED();
+    } catch (std::exception const& err) {
```

Comment:
Looks like `try/catch` do nothing special, so suggest to do next:
```
core.add_extension(getOVExtensionRelativePath());
```

Reply:
It was just for check what exception is thrown in the test, because it fails on CI